content/release-10-18+/applications-10-18+.md

```diff
@@ -387,6 +428,15 @@ Build version</th>
 <td>UI</td>
 </tr>
 
+<tr>
+<td>Web SDK</td>
+<td>In the data explorer, when deleting the min/max value of a data point, the Y-axis displayed "-1" and "1" instead of determining the maximum and minimum value based on the data. This has been fixed and the Y-axis shows the min/max value of the data again.</td>

```

Comment:
```suggestion
<td>In the data explorer, when deleting the min/max value of a data point, the Y-axis displayed "-1" and "1" instead of determining the maximum and minimum value based on the data. This has been fixed and the Y-axis shows the minimum and maximum values of the data again.</td>

```

Reply:
Min/max refers to the names in the UI which is why I left it in the context of the Y-axis and only used maximum/minimum in the more general context.